gnosis/eth/tests/eip712/test_eip712.py

```diff
@@ -86,7 +86,7 @@ def test_eip712_encode_hash(self):
         payload["types"] = self.types
         self.assertEqual(
             eip712_encode_hash(payload).hex(),
-            "7c02fe79823722257b42ea95720e7dd31d51c3f6769dc0f56a271800dd030ef1",
+            "0x7c02fe79823722257b42ea95720e7dd31d51c3f6769dc0f56a271800dd030ef1",
```

Comment:
This is related to return an HexBytes. I think that this issue should be tagged as breaking change.

Reply:
Totally agree